test/c/autostruct.yaml

```diff
@@ -5,6 +5,8 @@ directives:
   - sample_struct
   options:
     file: struct.c
+    clang:
+    - -DTEST_CACHE_MATCHING_WITH_CLANG_ARGS
```

Comment:
Spurious / incomplete change?

Reply:
Oh... Took me a bit.

Yeah, this would probably benefit from a commit message of its own if not a comment in the yaml / separate unit test. This is effectively part of a *different unit test* than the one it changes. Am I reading it right?